src/Makefile

```diff
@@ -256,6 +256,12 @@ endif
 
 ifeq ($(ARCH),general-64)
 	arch = any
+	# avoid apple clang error: "invalid arch name '-arch any'"
+	ifeq ($(KERNEL),Darwin)
+		arch = x86_64
+		sse = yes
+		sse2 = yes
+	endif
```

Comment:
can this be avoided in a different way? Probably general-64 should be specifying the architecture at all, like could this fail on M1?

Reply:
no problem on Intel-Mac

Reply:
@vondele good point! changed it to not specify arch at all on macos for general-64: https://github.com/official-stockfish/Stockfish/pull/3623/commits/c29077622e5a26f8ceab9c903192b27ee0727ec9